network/identity.go

```diff
@@ -64,19 +64,16 @@ func (i *identity) setup() {
 			peerID := conn.RemotePeer()
 			i.srv.logger.Trace("Conn", "peer", peerID, "direction", conn.Stat().Direction)
 
-			// limit by MaxPeers on incomming requests since we already limit
-			// the outgoing requests
```

Comment:
@munna0908 Do you have any idea why this comment is here? How do we limit outgoing requests?

Reply:
We check in run dial func [link](https://github.com/0xPolygon/polygon-sdk/blob/c0756d43fe3f921a9f45eaa52210c988f55b707b/network/server.go#L283), but our connection mechanism is 2 step process, we need to check in the later step aswell.